strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/app/spring/security/ConfigurationRepository.java

```diff
@@ -0,0 +1,5 @@
+package org.carlspring.strongbox.rest.app.spring.security;
+
+public class ConfigurationRepository {
```

Comment:
I'm not quite sure I get how you're using this empty class... Is Spring doing some weaving/instrumentation, or what?